tests/app/submitter/test_submitter.py

```diff
@@ -135,10 +135,29 @@ def test_when_message_sent_with_none_tx_id_then_tx_id_is_not_sent_in_header(self
         connection.channel.side_effect = Mock(return_value=channel)
         with patch('app.submitter.submitter.BlockingConnection', return_value=connection):
             # When
-            self.submitter.send_message(message={}, queue=self.queue_name, tx_id=None)
+            self.submitter.send_message(message={}, tx_id=None)
 
             # Then
             call_args = channel.basic_publish.call_args
             properties = call_args[1]['properties']
             headers = properties.headers
             self.assertEqual(len(headers), 0)
+
+
+class TestGCSSubmitter(TestCase):
+
+    def test_send_message(self):
+        with patch('app.submitter.submitter.storage.Client') as client:
+            submitter = GCSSubmitter(project_id='test_project', bucket_name='test_bucket')
+            published = submitter.send_message(message={'test_data'}, case_id='12345')
+
+            bucket = client.return_value.get_bucket.return_value
+            blob = bucket.blob.return_value
+
+            blob_name = bucket.blob.call_args[0][0]
+            assert blob_name == '12345'
```

Comment:
Minor nitpick but can we assertEqual here? The comparison object doesn't get shown in errors if we use assert.

Reply:
Could do, but I think we previously said that we were trying to move towards using asserts where possible. Can't quite remember. @bitdivision do you remember?

Reply:
@iwootten Indeed, one of the main advantages of pytest is that you don't need to use those methods anymore: https://docs.pytest.org/en/latest/assert.html

Not using the assert methods makes the code more readable and avoids nastiness around inheriting from testcase etc.